pkg/minikube/cluster/cluster.go

```diff
@@ -198,7 +198,7 @@ func (m *MachineConfig) CacheMinikubeISOFromURL() error {
 	defer response.Body.Close()
 
 	if response.StatusCode != http.StatusOK {
-		return fmt.Errorf("Received %d response from %s while trying to download minikube.iso", response.StatusCode, m.MinikubeISO)
+		return fmt.Errorf("Received %d response from %s while trying to download minishift.iso", response.StatusCode, m.MinikubeISO)
```

Comment:
One might think we are downloading minishift.iso, but it is still wrong. It's neither in this case.